src/validators/max.ts

```diff
@@ -0,0 +1,14 @@
+import { ValidatorFn } from './types'
+
+const maxValidator: ValidatorFn<number> = max => {
+  if (typeof max !== 'number') {
```

Comment:
I thought we were not checking type and let ts do its magic ?

Reply:
Consider that someone use our lib with javascript we need to check the type before ?

Reply:
Yeah, we had the same debate for the sdk, we said fuck it, we don't do it in casual js, why not do it in ts :)